core/src/main/resources/reference.conf

```diff
@@ -34,7 +56,7 @@ akka {
   }
 
   extensions = ${?akka.extensions}[com.romix.akka.serialization.kryo.KryoSerializationExtension,
-    de.heikoseeberger.constructr.ConstructrExtension, akka.cluster.pubsub.DistributedPubSub]
+    akka.management.cluster.bootstrap.ClusterBootstrap]
```

Comment:
@roarking we might be able to remove this since we are already initializing it manually